Backend/ChatAPI.Data/Models/ChatRoomUser.cs

```diff
@@ -0,0 +1,11 @@
+namespace ChatAPI.Data.Models;
+
+public class ChatRoomUser
+{
+    public int ChatRoomId { get; set; }
+    public int UserId { get; set; }
+    public string Description { get; set; }
+    
+    public List<ChatRoom> ChatRooms { get; set; }
```

Comment:
Do not use List for single related entiy, you have a property named as ChatRoomId and that is will be hoding Id of only one ChatRoom. So, there is won't be multiple record they have been related to ChatRoomUser.

Correct definition: `public ChatRoom ChatRoom { get; set; }`